client/my-sites/earn/launchpad/index.tsx

```diff
@@ -11,8 +18,45 @@ type EarnLaunchpadProps = {
 
 const EarnLaunchpad = ( { numberOfSteps, completedSteps }: EarnLaunchpadProps ) => {
 	const translate = useTranslate();
-
 	const site = useSelector( ( state ) => getSelectedSite( state ) );
+	const products: Product[] = useSelector( ( state ) => getProductsForSiteId( state, site?.ID ) );
```

Comment:
I have seen multiple useSelector calls written this way:
```const { products, connectedAccountId, stripeConnectUrl } = useSelector( state => ({
	products: getProductsForSiteId( state, site?.ID ),
	connectedAccountId: getConnectedAccountIdForSiteId( state, site?.ID ),
	stripeConnectUrl: getConnectUrlForSiteId( state, site?.ID ?? 0 )
}) ););
```

Reply:
Cleaner and works for me. I've updated this.